.ci/certs/generate-certs.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
These scripts are not used, right? (I mean this is one time thing and may be needed in the future but not part of the CI)

Reply:
They're not actively run no, they're more for posterity and so that if at some point we need to modify / regenerate them it's easier. ie. future me doesn't have to rediscover how they link together or what the right commands & config is 🥲

Reply:
Totally makes sense, thank you.